src/main/scala/magellan/Polygon.scala

```diff
@@ -190,6 +197,25 @@ class Polygon extends Shape {
 
   def getRing(index: Int): Int = indices(index)
 
+  def getRingPolygon(index: Int): Polygon = {
```

Comment:
why is this called ringPolygon?

Reply:
This a function getting the rings separately of a multipolygon. It did that because I was needed in the intersects function, but in the end I didn't used it.

Reply:
@Charmatzis can we remove this function if it is not needed?